libs/community/pyproject.toml

```diff
@@ -104,6 +104,7 @@ vdms = {version = "^0.0.20", optional = true}
 httpx-sse = {version = "^0.4.0", optional = true}
 pyjwt = {version = "^2.8.0", optional = true}
 oracledb = {version = "^2.2.0", optional = true}
+aerospike-vector-search = {version = "^0.6.1", optional = true}
```

Comment:
**Issue**: The new dependency 'aerospike-vector-search' is added as optional but without specifying that it should only be included in specific environments or setups, which might lead to unnecessary installation in production environments where it's not needed. <br> **Fix**: Include the 'aerospike-vector-search' dependency within a specific group that is only used in development or testing environments to prevent its installation in production. <br> **Code Suggestion**: 
 ```
 -aerospike-vector-search = {version = "^0.6.1", optional = true}
 +[tool.poetry.group.dev.dependencies]
 +aerospike-vector-search = {version = "^0.6.1"}
 ```